opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipCreationForm.js

```diff
@@ -190,11 +180,15 @@ const StixSightingRelationshipCreationForm = ({
                   </div>
                 </div>
                 <div className={classes.content}>
-                  <span className={classes.name}>
-                    {isMultipleFrom
-                      ? (<em>{t_i18n('Multiple entities selected')}</em>)
-                      : (defaultName(fromEntity))}
-                  </span>
+                  <Tooltip title={getMainRepresentative(fromEntity)}>
```

Comment:
same here

Reply:
As it's not present right now in the platform (both for relationships and sightings) and that the product didn't ask for it, I will get a tooltip only if there is one entity. Nothing will be displayed if there is more than one except for the "Multiple entities selected" to keep the same behavior.